src/DbUtils.php

```diff
@@ -853,6 +853,8 @@ public function getEntitiesRestrictCriteria(
             && $_SESSION['glpishowallentities']
         ) {
             return [];
+        } elseif ($value === []) {
```

Comment:
I wonder why we do not reach the previous condition; since `empty([])` is true - but I also don't know if we should.

Well, maybe someone else would have a better idea of what's expected here.

Reply:
In my case, I'm in a subentity, so `$_SESSION[‘glpishowallentities’]` is not true.